scripts/download.sh

```diff
@@ -42,11 +42,12 @@ else
   codecov_url="$codecov_url/${CODECOV_VERSION}"
   codecov_url="$codecov_url/${CODECOV_OS}/${codecov_filename}"
   say "$g ->$x Downloading $b${codecov_url}$x"
-  curl -O --retry 5 --retry-delay 2 "$codecov_url"
+  r="--retry 5 --retry-delay 2"
```

Comment:
you can probably pull this into `set_defaults` and use it in `validate.sh` also.

I would also name it something a little longer, `r`, might be used in a user's env vars already perhaps